src/components/articles.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+import { getAllArticles } from "../utils";
+
+const Articles = () => {
+  const [allArticles, setAllArticles] = useState([]);
+  useEffect(() => {
+    getAllArticles()
+      .then(({ articles }) => {
+        setAllArticles(articles);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }, []);
+  console.log(allArticles);
+  return (
+    <div className="articles_container">
+      {allArticles.map((article) => {
+        const regex = /\d+-\d+-\d+/;
+        const time = article.created_at.match(regex);
+        console.log(time);
```

Comment:
do we need regex for this can't be just display the date as a string which comes back?